base_module_record/models/base_module_record.py

```diff
@@ -26,7 +26,8 @@ class XElement(minidom.Element):
 
     def writexml(self, writer, indent="", addindent="", newl=""):
         writer.write(indent)
-        minidom.Element.writexml(self, writer, indent="", addindent="", newl="")
+        minidom.Element.writexml(self, writer, indent="", addindent="",
+                                 newl="")
         writer.write(newl)
```

Comment:
As this method is used to call to write respective xml file, Firsttime it would be called to add indentation in xml file. After, the further operation would be done and secondtime it would be called to add newline in xml file.